CHANGELOG_PENDING.md

```diff
@@ -17,6 +17,12 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
   - [abci/counter] \#6684 Delete counter example app
   - [txResults] \#9175 Remove `gas_used` & `gas_wanted` from being merkelized in the lastresulthash in the header
   - [abci] \#5783 Make length delimiter encoding consistent (`uint64`) between ABCI and P2P wire-level protocols
+  - [abci] \#9145 Removes unused Response/Request `SetOption` from ABCI (@samricotta)
+  - [abci] \#8656 Added cli command for `PrepareProposal`. (@jmalicevic)
+  - [abci] \#8901 Added cli command for `ProcessProposal`. (@hvanz)
```

Comment:
can we combine these lines

Reply:
I am preparing a CHANGELOG PR spanning all `PrepareProposal`/`ProcessProposal` changes that will deal with this